tools/package-tools/build-release.ts

```diff
@@ -48,16 +48,16 @@ export function composeRelease(buildPackage: BuildPackage) {
   copyFiles(packagesDir, 'README.md', releasePath);
   copyFiles(sourceDir, 'package.json', releasePath);
 
+  if (buildPackage.hasSchematics) {
```

Comment:
This should have a comment like "This must happen before..."

Reply:
Done. I've made it a bit more "generic" because `hasSchematics` is not specific to the Material schematics.